source/sync/configure/permissions.txt

```diff
@@ -34,11 +32,12 @@ to :ref:`partition-based-sync-permissions`.
    If your app does not use sync, check out :ref:`MongoDB Collection Rules
    <mongodb-rules>` for more information on rules for non-synced clusters.
 
+
```

Comment:
I might recommend adding a line here linking out to the guide (or whatever we end up calling it) - something like:

> For examples of how you might configure permissions for common data access patterns, refer to...